### PR TITLE
add photos to seeds

### LINKED ISSUE #495: Seed photos
You should be able to attach photos to seeds, as you do for plantings, harvests, etc.

### PR DESCRIPTION
**Pull request by maco**
Fixes #495 

untested, waiting on #1106 and #1107

### CODE REVIEW

**Comment by Br3nda** [comment]
Does the photo_mappings table not replace the photo_seeds photo_crops tables etc?

**Comment by maco** [comment]
@Br3nda after talking to @CloCkWeRX i learned that having a table that has fake foreign keys to multiple tables based on a column is a crappy database performance thing so we should stick to ordinary join tables. Thus, did not do `photo_mappings` after all.

**Comment by Br3nda** [comment]
@maco and  create table photo_mappings is still in this pr.

File: db/schema.rb [hunk 1]
```diff
@@ -373,6 +381,11 @@
     t.integer "planting_id"
   end
 
+  create_table "photos_seeds", id: false, force: :cascade do |t|
```
Comment (Br3nda) [inline db/schema.rb]: Should this table still be in here?

File: db/schema.rb [hunk 2]
```diff
@@ -355,6 +355,14 @@
     t.integer "product_id"
   end
 
+  create_table "photo_mappings", force: :cascade do |t|
```
Comment (Br3nda) [inline db/schema.rb]: Should this table still be here?

Comment (maco) [inline db/schema.rb]: oooo good catch

**Comment by CloCkWeRX** [comment]
Hey, crappy is too strong of a word. if we take a db centric view, its bad.
if we take an oop/orm view, its good. Was highlighting tradeoffs.

**Pull request merged**